bin/i-pi

```diff
@@ -83,7 +83,14 @@ if __name__ == '__main__':
 
     parser.add_option('-p', '--profile',
                       action='store_true', dest='do_yappi', default=False,
-                      help='Profile this run using Yappi.')
+                      help='Profile this run using yappi.')
+
+    parser.add_option('--profiler-clock', dest='yappi_clock', default="cpu",
+                      choices=['cpu', 'wall'],
+                      help='Type of profiling timing.')
```

Comment:
Change to more descriptive? e.g. Profiling timing from cpu time or wallclock time.